src/test/java/io/confluent/connect/jdbc/dialect/PostgreSqlDatabaseDialectTest.java

```diff
@@ -31,6 +35,16 @@
 
 public class PostgreSqlDatabaseDialectTest extends BaseDialectTest<PostgreSqlDatabaseDialect> {
 
+  @Override
+  @Before
+  public void setup() throws Exception {
+    super.setup();
```

Comment:
I think you should add this case to the base test, even though it makes the PR much larger

Reply:
I'm convinced.